docs/root/intro/version_history.rst

```diff
@@ -30,6 +30,8 @@ Version history
   restart Envoy. The behavior will not switch until the connection pools are recreated. The new
   circuit breaker behavior is described :ref:`here <arch_overview_circuit_break>`.
 * upstream: changed load distribution algorithm when all priorities enter :ref:`panic mode<arch_overview_load_balancing_panic_threshold>`.
+* router: added support for :ref:`regex_rewrite
```

Comment:
nit: alpha order

Reply:
Sorry, I didn't know this was supposed to be in alphabetical order.  I've moved it.